src/index.js

```diff
@@ -141,21 +151,37 @@ function mergeTailwindConfig(baseConfig) {
   return deepmerge(mergeConfig, baseConfig)
 }
 
-const postcssImportHyvaModules = ((opts = {}) => {
+/**
+ * PostCSS plugin to import Tailwind CSS files from Hyva modules.
+ *
+ * @typedef {Object} PostCSSPlugin
+ * @param {Object} opts - Options object.
+ * @param {string[]} [opts.hyvaModuleDirs=[]] - Directories to search for Hyva modules.
+ * @param {string[]} [opts.excludeDirs=[]] - Directories to exclude.
+ * @returns {Object} PostCSS plugin object.
+ */
+const postcssImportHyvaModules = (opts = {}) => {
+  const includeDirs = (opts.hyvaModuleDirs && setFullPaths(opts.hyvaModuleDirs)) || hyvaModuleDirs || [];
+  const excludeDirs = setFullPaths(opts.excludeDirs);
```

Comment:
Let's keep lines 164 and 165 consistent - either checking if the property exists before calling `setFullPath` or not.

Reply:
Consider it done, but it should not be needed since the setFullPaths already returns an empty array if the input is empty/null